transformers/progress_report_resource.go

```diff
@@ -0,0 +1,35 @@
+package transformers
+
+import (
+	"github.com/companieshouse/insolvency-api/models"
+	"github.com/companieshouse/insolvency-api/utils"
+)
+
+// ProgressReportResourceRequestToDB transforms a progress report request to a dao model
+func ProgressReportResourceRequestToDB(req *models.ProgressReport) *models.ProgressReportResourceDao {
+	etag, err := utils.GenerateEtag()
```

Comment:
Check if err is not nil and log is important here

Reply:
done